bigquery/indexer.py

```diff
@@ -117,26 +95,7 @@ def _rows_from_export(
         blob.delete()
 
 
-# Sample and participant tables need to be indexed differently.
-# For participant tables, we can use partial updates
-# (https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update.html#_updates_with_a_partial_document)
-#
-# If one participant table has weight and another has height:
-# - First weight table is indexed. Participant documents get a weight field.
-# - Then height table is indexed. Participant documents get a height field. The
-#   weight fields are unchanged.
-#
-# Now say one sample table has center and another has platform.
-# - First center table is indexed. For each participant document, a samples
-#   field is created. The samples field contains an array of nested objects,
-#   each of which has a center field.
-# - Then platform table is indexed. For each participant document, the samples
-#   field is overwritten to the new value, which contains platform and not
-#   center.
-# In order to keep the center field, one must use a script. See
-# https://discuss.elastic.co/t/updating-nested-objects/87586/2 and
-# https://www.elastic.co/guide/en/elasticsearch/reference/6.4/docs-update.html
-def _sample_scripts_by_id_from_export(
+def _sample_docs_by_id_from_export(
```

Comment:
Its not so much that I'm converting it into a dict, its more so that I'm adding the data to the existing global dict. I think changing these methods to return dicts doesn't do much good.

I think what would I could do is pass the global dict to these methods, and update it there. That would save a lot of code. I'll go ahead and do that.